backend/libinput/pointer.c

```diff
@@ -141,6 +141,54 @@ void handle_pointer_axis(struct libinput_event *event,
 	wlr_signal_emit_safe(&wlr_dev->pointer->events.frame, wlr_dev->pointer);
 }
 
+void handle_pointer_axis_wheel(struct libinput_event *event,
+		struct libinput_device *libinput_dev) {
+	struct wlr_input_device *wlr_dev =
+		get_appropriate_device(WLR_INPUT_DEVICE_POINTER, libinput_dev);
+	if (!wlr_dev) {
+		wlr_log(WLR_DEBUG, "Got a pointer event for a device with no pointers?");
+		return;
+	}
+	struct libinput_event_pointer *pevent =
+		libinput_event_get_pointer_event(event);
+	struct wlr_event_pointer_axis wlr_event = { 0 };
+	wlr_event.device = wlr_dev;
+	wlr_event.time_msec =
+		usec_to_msec(libinput_event_pointer_get_time_usec(pevent));
```

Comment:
i don't know the full context here, but why get usec when you cast to msecs anyway? just use `foobar_get_time()` instead?

Reply:
This function is 99% copied from the existing `handle_pointer_axis` code so I didn't want to rock the boat too much.